syntax = 'proto3';


message ImageToRecognize {
  bytes data = 1;
}

message RecognizedResult {
  string result = 1;
}

service Recognizer {
    rpc RecognizeFormula(ImageToRecognize) returns (RecognizedResult) {}
    rpc RecognizeMixed(ImageToRecognize) returns (RecognizedResult) {}
    rpc RecognizeText(ImageToRecognize) returns  (RecognizedResult) {}
}
